package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "resin-device-config",
-  "version": "3.2.0",
+  "version": "3.3.0",
```

Comment:
Usually the package version should be bumped by the maintainer after the PR is merged